client/dashboard/app/context.tsx

```diff
@@ -16,6 +16,10 @@ export type MeSupports = {
 	apps: boolean;
 };
 
+export type OnboardingLink = {
+	href: string;
+};
```

Comment:
In the future I suspect we'll want to customise the link labels too, so leaving space for that.